lxd/vm_qemu.go

```diff
@@ -314,6 +314,37 @@ func (vm *vmQemu) getStoragePool() (storagePools.Pool, error) {
 	return vm.storagePool, nil
 }
 
+func (vm *vmQemu) eventHandler() func(event string, data map[string]interface{}) {
```

Comment:
@stgraber please could you call this monitorEventHandler() so its clear what sort of events is it handling, it also helps to show its related to the getMonitorPath() function. Perhaps for consistency `getMonitorPath` should be renamed to `monitorPath()` too, or `getMonitorEventHandler()` might actually be more appropriate because it itself is not the event handler, it returns an event handler function (this caught me out first pass).

Reply:
Will do